flexmeasures/data/scripts/clean_database.sh

```diff
@@ -13,6 +13,14 @@ function is_database() {
 function create_database() {
  echo "Creating a new database ..."
  sudo -i -u postgres createdb -U postgres $1
+ if [[ -n "$2" ]]; then
```

Comment:
These lines should be in their own function `create_user_and_give_privileges`.

Reply:
i have made two separate functions `create_user` and `grant_privileges`